src/gui/src/settings/options-window.cpp

```diff
@@ -283,6 +287,16 @@ OptionsWindow::OptionsWindow(Profile *profile, ThemeLoader *themeLoader, QWidget
 	ui->comboImagePositionVideoH->setCurrentIndex(positionsH.indexOf(settings->value("imagePositionVideoH", "left").toString()));
 	ui->lineImageBackgroundColor->setText(settings->value("imageBackgroundColor", QString()).toString());
 
+	ui->comboActionButtons->clear();
+	ui->comboActionButtons->addItem("All", ZoomWindow::ButtonVisibility::All);
+	ui->comboActionButtons->addItem("None", ZoomWindow::ButtonVisibility::None);
+	ui->comboActionButtons->addItem("Favorites", ZoomWindow::ButtonVisibility::Favorites);
+	ui->comboActionButtons->addItem("NonFavorites", ZoomWindow::ButtonVisibility::NonFavorites);
+	ui->comboActionButtons->setCurrentIndex(settings->value("actionButtons", "All").toInt());
```

Comment:
Shouldn't the default be `ZoomWindow::ButtonVisibility::All` instead? Otherwise you're casting a string here.